packages/nc-gui/components/smartsheet/details/Api.vue

```diff
@@ -95,7 +97,7 @@ const code = computed(() => {
 const api = new Api({
   baseURL: "${(appInfo.value && appInfo.value.ncSiteUrl) || '/'}",
   headers: {
-    "xc-auth": ${JSON.stringify(token.value as string)}
+    "xc-token": "<CREATE YOUR API TOKEN FROM : https://hostname.com/#/account/tokens>"
```

Comment:
By mentioning hostname.com - i mean this has to be dynamic and we have to figure it out in the frontend.

Reply:
You can use something like `location.origin + location.pathname`

```
    "xc-token": `<CREATE YOUR API TOKEN FROM : ${location.origin + location.pathname}/#/account/tokens>`
```